compiler-rs/clients_schema_to_openapi/src/paths.rs

```diff
@@ -311,6 +313,25 @@ fn get_path_parameters(template: &str) -> Vec<&str> {
     result
 }
 
+fn split_summary_desc(desc: String) -> SplitDesc{
+    let mut parts = desc.split(['.','\n',':']);
+    let first_line = parts.next().unwrap_or_else(|| "");
+    
+    let new_desc = desc.replace(first_line,"");
+    let trim = new_desc.trim();
+    let remove_period = trim.strip_prefix('.').unwrap_or_else(|| trim);
+    let remove_column = remove_period.strip_prefix(':').unwrap_or_else(|| remove_period);
+    SplitDesc {
+        summary: String::from(first_line.trim()),
+        description: String::from(remove_column.trim())
+    }
+}
+
+struct SplitDesc {
+    summary: String,
+    description: String
```

Comment:
These should be `Option<String>`. There are lots of single-sentence descriptions in `schema.json` that result in a description that is `Some("")` while it should be `None` and not output in the resulting OpenAPI schema.